syft/pkg/cataloger/cpp/cataloger.go

```diff
@@ -10,5 +10,6 @@ const catalogerName = "conan-cataloger"
 func NewConanCataloger() *generic.Cataloger {
 	return generic.NewCataloger(catalogerName).
 		WithParserByGlobs(parseConanfile, "**/conanfile.txt").
-		WithParserByGlobs(parseConanlock, "**/conan.lock")
+		WithParserByGlobs(parseConanlock, "**/conan.lock").
+		WithParserByGlobs(parseConaninfo, "**/conaninfo.txt")
 }
```

Comment:
it seems that the conaninfo.txt fits a different use case than the other existing parsers. The existing parsers seem best fit for detecting declarations of software to be installed where as the new conaninfo parser detects evidence of installed software. @Pro  is that interpretation right?

If so, then we should probably leave the existing cataloger as is and make a new `NewConanInfoCataloger()` (or another name) that houses your new parser. That way we could keep the existing cataloger for directory scans and the new cataloger for image scans.

Reply:
@wagoodman your interpretation is fully correct!

I changed the code to have a separated ConanInfoCataloger.

Can you please check the changes?